### ISSUE TITLE
Read multiple house types in a DXF file

### ISSUE THREAD

**Issue by yixingDBJ** [haplokuon/netDxf#170]
![图片](https://user-images.githubusercontent.com/44353513/76192212-3f5eb880-621c-11ea-9d05-0c79f3be72fa.png)
Hello, how about the DXF file like the picture? Can you identify several house types? If you can, which variable or method in dxfdocument

**Comment by haplokuon** [comment]
Read [this](https://github.com/haplokuon/netDxf/issues/165), same problem.

**Comment by yixingDBJ** [comment]
Thank you for your reply, but I may not have expressed my question clearly. For example, the result I want to get is that there is no way to automatically identify that it contains six independent parts, each part is a complete CAD picture, but they are placed on the same large picture

**Comment by yixingDBJ** [comment]
![图片](https://user-images.githubusercontent.com/44353513/76288703-4cde7600-62e2-11ea-983a-59e0165adc8f.png)
At last, the effect of recognition is equivalent to that each of them is independent

**Comment by haplokuon** [comment]
No. What to do with the contents of the DXF is up to you, but if you know the axis aligned bounding box of each part you can check which entities lies inside them.
